sdk/mongocluster/arm-mongocluster/review/arm-mongocluster-api.api.md

```diff
@@ -7,82 +7,25 @@
 import { Client } from '@azure-rest/core-client';
 import { ClientOptions } from '@azure-rest/core-client';
 import { HttpResponse } from '@azure-rest/core-client';
-import { OperationOptions } from '@azure-rest/core-client';
-import { OperationState } from '@azure/core-lro';
 import { Paged } from '@azure/core-paging';
-import { PollerLike } from '@azure/core-lro';
 import { RawHttpHeaders } from '@azure/core-rest-pipeline';
 import { RequestParameters } from '@azure-rest/core-client';
 import { StreamableMethod } from '@azure-rest/core-client';
 import { TokenCredential } from '@azure/core-auth';
 
 // @public
-export function createMongoClusterManagement(credential: TokenCredential, options?: MongoClusterManagementClientOptionalParams): DocumentDBContext;
+export function createMongoClusterManagement(credential: TokenCredential, options?: MongoClusterManagementClientOptions): DocumentDBContext;
 
 // @public (undocumented)
 export type DocumentDBContext = Client & {
     path: Routes;
 };
 
 // @public
-export function firewallRulesCreateOrUpdate(context: DocumentDBContext, subscriptionId: string, resourceGroupName: string, mongoClusterName: string, firewallRuleName: string, resource: FirewallRule_2, options?: FirewallRulesCreateOrUpdateOptionalParams): PollerLike<OperationState<FirewallRule_2>, FirewallRule_2>;
-
-// @public
-export function firewallRulesDelete(context: DocumentDBContext, subscriptionId: string, resourceGroupName: string, mongoClusterName: string, firewallRuleName: string, options?: FirewallRulesDeleteOptionalParams): PollerLike<OperationState<void>, void>;
-
-// @public
-export function firewallRulesGet(context: DocumentDBContext, subscriptionId: string, resourceGroupName: string, mongoClusterName: string, firewallRuleName: string, options?: FirewallRulesGetOptionalParams): Promise<FirewallRule_2>;
-
-// @public
-export function firewallRulesListByMongoCluster(context: DocumentDBContext, subscriptionId: string, resourceGroupName: string, mongoClusterName: string, options?: FirewallRulesListByMongoClusterOptionalParams): PagedAsyncIterableIterator<FirewallRule_2>;
-
-// @public
-export interface MongoClusterManagementClientOptionalParams extends ClientOptions {
+export interface MongoClusterManagementClientOptions extends ClientOptions {
```

Comment:
there's no client level operations in this case, so it actually have no api exported in the top level api subpath.